src/main/java/magicbees/main/Config.java

```diff
@@ -332,6 +342,31 @@ private void doGeneralConfigs() {
                 32767);
         thaumcraftNodeMaxSize = p.getInt(256);
 
+        p = configuration.get(
+                CATEGORY_GENERAL,
+                "drainerTimeBetween",
+                200,
+                "The time in ticks between Apiamancer's Drainer essentia generation",
```

Comment:
Spelling: Apiamancer -> Apimancer

Reply:
But the block is named the Apiamancer's Drainer.

Reply:
![image](https://github.com/GTNewHorizons/MagicBees/assets/8459776/c80b52f0-7ea5-4364-ac5e-7ed4e096f44f)

Reply:
It's quite annoying to not be able to find something in NEI because it's not spelled as expected.